.SRCINFO

```diff
@@ -1,6 +1,6 @@
 pkgbase = ros-noetic-libfranka
 	pkgdesc = C++ library for Franka Emika research robots
-	pkgver = 0.7.1
+	pkgver = 0.8.0
```

Comment:
While `libfranka` has released 0.8.0, the ROS package's (`ros-noetic-libfranka`) latest release is still at 0.7.1. If you would like to use `libfranka`'s release I recommend you create a separate AUR package for it. This PKGBUILD follows the latest ros release of libfranka not the pure C++ libfranka library.